Aliucord/src/main/java/com/aliucord/settings/Crashes.java

```diff
@@ -68,10 +74,60 @@ public void onViewBound(View view) {
         setActionBarTitle("Crash Logs");
 
         Context context = requireContext();
-        LinearLayout layout = (LinearLayout) ((NestedScrollView) ((CoordinatorLayout) view).getChildAt(1)).getChildAt(0);
-
         int p = Utils.getDefaultPadding() / 2;
-        layout.setPadding(p, p, p, p);
+        File folder = new File(Constants.BASE_PATH, "crashlogs");
+        File[] files = folder.listFiles();
```

Comment:
you should move this into the clear button onClick handler and use folder.exists() instead to determine whether the folder exists.

Also I'm pretty sure calling button.setClickable(true) does nothing since that's the default, so you should either use button.setClickable(folder.exists()) or just don't add the button in the first place